meta/netbeansrelease.json

```diff
@@ -666,7 +666,7 @@
     },"release1303": {
         "position": "18",
         "ant": "ant_latest",
-        "jdk": "jdk_11_latest",
+        "jdk": "jdk_1.8_latest",
```

Comment:
I think this is more accurate, especially when [netbeans-vscode](https://ci-builds.apache.org/job/Netbeans/job/netbeans-vscode/) job still uses JDK8.